src/jsp/jsp_cl.cpp

```diff
@@ -1063,6 +1099,25 @@ drop_stored_procedure (const char *name, SP_TYPE_ENUM expected_type)
       goto error;
     }
 
+  // delete _db_stored_procedure_code
+  err = db_get (sp_mop, SP_ATTR_TARGET, &target_val);
+  if (err != NO_ERROR)
+    {
+      goto error;
+    }
+
+  target = db_get_string (&target_val);
+  class_name = get_class_name (target);
```

Comment:
class_name, it's not used currently. Is it left to use in the future (commentd as TODO) ?

Reply:
I've moved the line inside of the TODO comment block.